code/elemental/ElementalSearchableFieldsBlock.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace SilverStripe\FrameworkTest\Elemental\Model;
+
+use DNADesign\Elemental\Models\BaseElement;
+use SilverStripe\Forms\SearchableDropdownField;
+use SilverStripe\Forms\SearchableMultiDropdownField;
+
+class ElementalSearchableFieldsBlock extends BaseElement
+{
+    private static $table_name = 'ElementalSearchableFieldsBlock';
```

Comment:
```suggestion
    private static $table_name = 'ElementalSearchableFieldsBlock';
    
    private static string $singular_name = 'SearchableFields Block';

    private static string $plural_name = 'SearchableFields Blocks';
```
Otherwise it just shows up as "block"
![image](https://github.com/silverstripe/silverstripe-frameworktest/assets/36352093/6858764b-63b0-43ab-80a8-93978ac905c6)

Reply:
And before you say that doesn't matter 'cause no one uses this directly, I have frameworktest installed in nearly every environment I spin up to take advantage of its fixtures and boilerplate, so it matters to me. :p

Reply:
Updated